src/App.js

```diff
@@ -1,26 +1,11 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
```

Comment:
- [ ] As it is a requirement for this project, your App.js component is supposed to be class-based. Please kindly fix that. Screenshot of the requirement below.

 ![alex](https://user-images.githubusercontent.com/65852621/143157265-9ecec0c5-089a-43dc-815c-447148818fad.png)

Reply:
Hello, thank you for your suggestion, I was thinking the components only are the files inside the components folder, in this case, Calculator.js, I already refactor the function!